src/sync/installation.ts

```diff
@@ -62,7 +62,11 @@ export const sortedRepos = (repos: Repositories): [string, RepositoryData][] =>
 			new Date(a[1].repository?.updated_at || 0).getTime()
 	);
 
-const getNextTask = async (subscription: Subscription): Promise<Task | undefined> => {
+const getNextTask = async (subscription: Subscription, jiraHost: string): Promise<Task | undefined> => {
```

Comment:
instead of passing in the `jiraHost`, it might make more sense to pass in the boolean of `includeBuildAndDeployments`

Reply:
it makes more sense to pass the boolean, it just got awkward with line 
162-`} else if (!(await getNextTask(subscription, jiraHost))) {`
	
Could have FF evaluating in 2 places(above each getNext) but seemed convoluted to evaluate a flag that might not get called....

If I wanted to evaluate it once, it got pretty messy fast passing it all over the show.....

Reply:
yeah, fair.